sdk/cosmosdb/cosmos/src/client/SasToken/ControlPlanePermissionScope.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I started refactoring this file and I think its just not necessary at all. I was able to remove it and use PermissionScopes file directly https://github.com/Azure/azure-sdk-for-js/commit/ab736c194b2f8b6a2ec890cdbc91c4e55d071785